app/templates/security.hbs

```diff
@@ -0,0 +1,52 @@
+<PageHeader @title='Security Information' />
+
+<TextContent @boxed={{true}}>
+
+  <h2 id='crates-io-security'>Security of crates.io itself</h2>
+
+  <p>Safety is one of the core principles of Rust, and to that end, we would like to ensure that cargo, crates.io, docs.rs, and
+    related tools have secure implementations. To disclose security vulnerabilities in the crates.io service itself (as opposed
+    to crates hosted on crates.io) or any other <a href='https://github.com/rust-lang'>repository in the rust-lang
+    organization</a>, please follow the <a href='https://www.rust-lang.org/policies/security'>Rust Security policy</a>.</p>
+
+    <p>Thank you for taking the time to responsibly disclose any issues you find.</p>
+
+  <h2 id='crate-security'>Security of crates hosted on crates.io</h2>
+
+  <p>To disclose security vulnerabilities found in a crate that is hosted on crates.io, seek guidance from the individual crate's
+    owners and their specific policies. Commonly, projects include a file named <code>SECURITY.md</code> that contains the
+    crate's security policies and procedures.</p>
```

Comment:
An extra bit of nuance is that we do want to hear about malicious crate activity, where the owner is likely complicit. That should probably be linked to the usage policy, which does mention this, though not all usage violations are security issues.

Reply:
Yeah, I thought about trying to clarify "intentional" vs "unintentional" security vulnerabilities... maybe I can say malware? Edit incoming.